In [3]:
df = pd.read_csv('crime_data.csv')
df.shape

(690454, 28)

The original dataset contains 690454 observations and 28 columns, which is quite a lot of data, so I split the dataset into two smaller parts (each containing 5000 rows) for further training and testing the ML model.

In [4]:

import pandas as pd
part1 = df.iloc[:5000, :]
part2 = df.iloc[5000:10000, :]

In [5]:
part1.to_csv('data_0.csv', index=False)
part2.to_csv('data_1.csv', index=False)